lib/collections/transform/cartOrder.js

```diff
@@ -46,26 +47,23 @@ function getSummary(items, prop, prop2, shopId) {
 }
 
 /**
- * Reaction transform collections
- *
- * transform methods used to return cart and order calculated values
- * count, subTotal, shipping, taxes, total
- * are calculated by a transformation on the collection
- * and are available to use in template as cart.xxx or order.xxx
- * in template: {{cart.getCount}}
- * in code: Cart.findOne().getTotal()
+ * Reaction transform methods on Collections
+ * @file Use transform methods to return Cart and Order calculated values: count, subTotal, shipping, taxes, total. Use these methods on Cart and Orders in templates, `{{cart.getCount}}` and also in code, `Cart.findOne().getTotal()`. These are calculated by using a Mongo Collection's {@link http://docs.meteor.com/api/collections.html#Mongo-Collection transform functionality}.
```

Comment:
@zenweasel Is this a good blurb for the `@file`? It shows up at the top of the page. Edits encouraged!

Reply:
Can we format this to fit within the 120 line width? I can't read all of it here as it flows off the page

Reply:
@machikoyasuda needs to follow our standard code guidelines.